worlds/wargroove2/Levels.py

```diff
@@ -0,0 +1,416 @@
+from typing import List
+
+from BaseClasses import Region, Entrance, MultiWorld
+from .Locations import location_table, Wargroove2Location
+from worlds.generic.Rules import set_rule
+
+region_names: [str] = ["North 1", "East 1", "South 1", "West 1",
```

Comment:
```suggestion
region_names: List[str] = ["North 1", "East 1", "South 1", "West 1",
```
gonna be nitpicky about typing this review, fair warning. As always, these are recommendations, not hard requirements. But also I like when it looks cleaner so